src/model/entities/action.ts

```diff
@@ -58,9 +58,9 @@ export enum MutateActionType {
   APPLY_ARGUMENTS = 'APPLY_ARGUMENTS'
 }
 
-export interface MutateActionWithPieceMethods {
+export interface MutateActionWithOnAirAndNextPiecesMethods {
   type: MutateActionType.PIECE
-  updateActionWithPiece: (action: Action, piece: Piece) => Action
+  updateActionWithPiece: (action: Action, onAirPiece: Piece | undefined, nextPiece: Piece | undefined) => Action
```

Comment:
```suggestion
  updateActionWithPiece: (action: Action, onAirPiece?: Piece, nextPiece?: Piece) => Action
```
Just another suggestion 😄

Reply:
Updated